instrumentation/README.md

```diff
@@ -52,8 +52,10 @@ For example:
 - `go.opentelemetry.io/contrib/instrumentation/gopkg.in/macaron.v1/otelmacaron`
 - `go.opentelemetry.io/contrib/instrumentation/database/sql/otelsql`
 
-Exceptions to this rule do exist.
-For example, the [runtime](./runtime) instrumentation does not instrument a Go package and does not fit this structure.
+The following packages are an exception to this rule as they do not instrument a Go package:
+
+- [host](./host)
+- [runtime](./runtime)
```

Comment:
I think the original reads better. Can we add the `host` package using the same sentence structure?

```suggestion
Exceptions to this rule exist.
For example, the [runtime](./runtime) and [host](./host) instrumentation do not instrument any Go package and do therefore do not fit this structure.
```